include/nds/arm9/math.h

```diff
@@ -319,6 +319,16 @@ static inline uint32_t sqrt64(uint64_t a)
     return sqrt64_result();
 }
 
+
+/// 32 bit floating point square root
+///  @param valid 32 bit positive floating point value.
+///  @warning Subnormals are treated as if they were zero.  NaN and Inf are not handled. 
+///  @return 32 bit floating point value
+///
```

Comment:
```
/// 32 bit floating point square root.
///
/// @warning Subnormals are treated as if they were zero. NaN and Inf are not handled. 
///
/// @param valid 32 bit positive floating point value.
/// @return 32 bit floating point value.
float hw_sqrtf(float x);
```

Don't add empty lines before the comment, and don't add many lines after the function declaration, just one.

Also, I'm not supre sure about the name of the function, but I don't have good ideas. If someone wants to propose something...